src/commands/linkArticles.ts

```diff
@@ -83,7 +87,39 @@ async function insert(
 	if (templateString.includes("{description}") && !article.description)
 		new Notice("The article has no description.");
 
+	if (template.createArticleNote) {
+		const createdNote = await createNoteInFolder(app, noteTitle, insert, template);
+		if (createdNote != null) {
+			insert = `[${noteTitle}](${createdNote})`;
```

Comment:
This is not how linking to notes with a path works in Obsidian. It's `[[<path>|<name>]]` (Try it out by creating two files with the same name and reference both. Then view the links in the src view). 
This also should match the new link format setting 
![image](https://github.com/StrangeGirlMurph/obsidian-wikipedia-search/assets/62220780/19b95f4b-3d0d-4c27-a51a-11f7c62e764d)

Reply:
yeah, thats my fault ^^ misplaced that with standard markdown.